### ISSUE TITLE
How can I use the implicit login registration page from jhipster ( spring boot,)  instead of keycloak login window?

### ISSUE THREAD

**Issue by dkarthi1973** [dsb-norge/vue-keycloak-js#50]
I have tried to register new user from Jhipster (Oauth2, keycloak)  but getting authorization error.( based on bearer token ). If any reference, please let us know.

**Comment by nlien** [comment]
Hi there. How is this an issue concerning this plugin?
I am not aware of any JHipster scaffolding that uses this plugin, and if so, it would be hard to follow up your issue without any indications of what is wrong here. No error messages, stacktraces, reproducing steps... 

Despite being actively involved in the continuation of this plugin, I'm inclined to propose closing this issue.

**Comment by anderius** [comment]
I am closing this issue due to lack of feedback.
